tests/Unit/Elasticsearch/Document/DocumentCollectionTest.php

```diff
@@ -5,33 +5,33 @@
 use EMS\CommonBundle\Elasticsearch\Document\DocumentCollection;
 use EMS\CommonBundle\Elasticsearch\Response\ResponseInterface;
 use EMS\CommonBundle\Elasticsearch\Document\DocumentInterface;
-
 use PHPUnit\Framework\TestCase;
 
 class DocumentCollectionTest extends TestCase
 {
-    /** @var \ResponseInterface&\PHPUnit\Framework\MockObject\MockObject $mockResponse */
-    private $mockResponse;
+    /** @var letypehint $mockR */
```

Comment:
what is "letypehint"? should this not be ResponseInterface|MockObject?
```suggestion
    /** @var ResponseInterface|PHPUnit\Framework\MockObject\MockObject $mockResponse */
```